examples/ingress-recommendations/README.md

```diff
@@ -0,0 +1,175 @@
+# Ingress recommendations
+
+This example scenario showcases the BotKube recommendations feature around Ingress resources.
+
+## Prerequisites
+
+### Software
+
+Install the following applications:
+
+- [colima](https://github.com/abiosoft/colima)
+- [Helm](https://helm.sh/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Mkcert](https://github.com/FiloSottile/mkcert)
+
+### Set up local Kubernetes cluster
+
+Run the command:
+
+```bash
+colima start --runtime containerd --kubernetes --cpu 2 --memory 3 --disk 20 --profile botkube
+```
+
+### Install Ingress Nginx Controller
+
```

Comment:
Do you think we should state that they need to go to the project root folder due to example folder dependency?